packages/server/package.json

```diff
@@ -56,6 +56,7 @@
     "dayjs": "^1.9.3",
     "debug": "^4.3.2",
     "duplexify": "4.1.1",
+    "electron": "21.0.0",
```

Comment:
I think this is intentionally a devDep - we bundle electron in later during the build step, and it's declared in the top level package.json. I *think* adding this here as a regular dep will bloat the binary with an additional electron (need to check this).

Why do we need to add it here?

Reply:
The reason why we're seeing error
```
Cannot find module 'electron'
```
is that in the binary file, in MacOS for example, in `/Library/Caches/Cypress/10.11.0/Cypress.app/Contents/Resources/app/node_modules`, there does not exist the needed `electron` package (you can check in existing released apps), so either we somehow fix it later in the build step to include that package, or we define it in deps.
I do see the CI checks failed because of a larger zip size, so definitely we can look into this more. But either way, the binary size would probably increase.

Reply:
Yep, CI check: https://app.circleci.com/pipelines/github/cypress-io/cypress/45714/workflows/902afa25-43af-4183-bdf4-e3348df80565/jobs/1922692?invite=true#step-113-91

Reply:
I will revert this and see how we go.